src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(number) {
+      calculator.result += number;
+
+      return this;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
+
+      return this;
+    },
+
+    multiply(number) {
+      calculator.result *= number;
+
+      return this;
+    },
+
+    divide(number) {
+      calculator.result /= number;
+
+      return this;
+    },
```

Comment:
Instead of having every callback edit the final `result` property you could simply return the result of the operation and update the `operate` callback to set result `this.result = operation(number)`.

This better follows the single responsibility principle by centralizing the `result` value update handling in `operate` and allowing each callback to simply perform the operation it's supposed to.